README.md

```diff
@@ -508,6 +512,107 @@ rollback = true
  }}
 ```
 
+#### With a `YAML` file
```

Comment:
What if we rename it to "From a `YAML` file"? The title would be the same as the function `from_yaml`.